### PR TITLE
Watching APIs

### PR DESCRIPTION
**Pull request by sowmyaraj92**
Enable watch for APIs and receive in-app/email notifications whenever changes are made.

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 1]
```diff
@@ -103,6 +104,34 @@ To create a collection based on your schema, click **Generate Collection**. You
 
 You can also configure how the collection should be generated by clicking **Show advanced settings**.
 
+## Watching an API
+
+The watch option allows you to receive an email/in-app notification when your team member belonging to the same workspace modifies the API like updating the schema, attaching an element to the API, adding a comment to the API and so on.
```
Comment (SueSmith) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:109]: I'd change "like" to "for example"

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 2]
```diff
@@ -103,6 +104,34 @@ To create a collection based on your schema, click **Generate Collection**. You
...
+
+Once you've created the API, click the __watch__ option to start watching the API.
+
+![Watch Option](https://assets.postman.com/postman-docs/watch-option-active.jpg)
+
+After enabling the watch option, you will get a pop-up notification stating "__You are watching this API__."
+
+![Watch API](https://assets.postman.com/postman-docs/watch-api.jpg)
+
+Being a watcher, you will be notified whenever the API schema is modified. Imagine you have an account A where you have enabled the watch option. Now, from a different account B present in the same team workspace, you update the schema. After updating, click the __save__ button to save the changes.
```
Comment (SueSmith) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:119]: I'd reword the bit from "Imagine" up to just before "Click the bell..." in the next paragraph along these lines:

"If you're watching an API in your current team workspace, you will see notifications whenever changes are made to the schema in Postman even if you are logged into a different account."

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 3]
```diff
@@ -103,6 +104,34 @@ To create a collection based on your schema, click **Generate Collection**. You
...
+
+Now, you will receive a notification for account A. Click the bell icon on the top right corner to view the notification. The popup will indicate further information about the updated schema.
+
+![Notification Watch API](https://assets.postman.com/postman-docs/notification-watch-api.jpg)
+
+In addition to this, you will receive an email with the information regarding who has made the change, what was the change and when was the change made.
```
Comment (SueSmith) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:125]: last part > "what the change was, and when it was made."

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 4]
```diff
@@ -103,6 +104,34 @@ To create a collection based on your schema, click **Generate Collection**. You
...
+
+![Email Notification](https://assets.postman.com/postman-docs/email-notification.jpg)
+
+Click the __View Schema__ button to view the API in the web dashboard.
+
+![View Schema](https://assets.postman.com/postman-docs/watch-api-view-schema.jpg)
+
+> Note: If you have created the API and modified the schema from your account, then you will not be receiving the email/in-app notification for the changes made.
```
Comment (SueSmith) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:133]: We can lose the "Note:" text, as long as it's in a blockquote that indicates it's a note so you can jump straight into the info.. 

I'd also reword a tiny bit: "If you created the API and modified the schema from the same account, you will not receive email or in-app notifications for changes made."

**Review by SueSmith — CHANGES_REQUESTED** [review]
looking good @sowmyaraj92! few wee daft requests and if we can sort the package lock stuff.. 😭

**Comment by sowmyaraj92** [comment]
Moved to a new branch `watch-api`.
